timemachine/fe/model_rabfe.py

```diff
@@ -397,8 +400,8 @@ def simulate_futures(
         all_futures = []
         sys_params, model, futures = self._futures_a_to_b(
             ff_params,
-            mol_a,
-            mol_b,
+            mol_a,  # ACTUAL_MOL
+            mol_b,  # BLOCKER
```

Comment:
Could this be reflected in variable names instead?

Reply:
I'd rather remove the comments here (they're extraneous), and semantically the class shouldn't care about which is which.